### ISSUE TITLE
Group chat unable to change topic

### ISSUE THREAD

**Issue by KamalGalrani** [bitlbee/bitlbee-facebook#121]
I am unable to change topic using /topic in group chat channel. Every time I try facebook reconnects saying `Error: Failed generic API operation`

**Comment by dequis** [comment]
This works for me. I'm going to need some debug output

**Comment by KamalGalrani** [comment]
I have attached the log. All I did was connect to facebook then list chats using fbchats, join using fbjoin then change topic using /topic

[bitlbee.txt](https://github.com/bitlbee/bitlbee-facebook/files/721645/bitlbee.txt)

**Comment by dequis** [comment]
Thanks!

I forgot to mention this tends to have private stuff. I edited your comment to only include the relevant part and removed the access token. You might want to invalidate that using 'end session' in [this page](https://www.facebook.com/settings?tab=security&section=sessions&view)

I wonder how linear the x-fb-rev parameter is. I'm in 2790673, you're in 2789434 and it works here. I switched to beta and got 2789434 and still works. Maybe it really is just a case of "Service temporarily unavailable", unlike the other ticket. The bug here is mainly failing so hard, which is something this plugin likes to do.

**Comment by KamalGalrani** [comment]
I thought it might contain private info so created an account specifically for this purpose as I didn't want to care what part is private.

The second para goes over my head. Can you explain it to me in simpler terms

**Comment by dequis** [comment]
>I thought it might contain private info so created an account specifically for this purpose as I didn't want to care what part is private.

Nice. I like you.

>The second para goes over my head. Can you explain it to me in simpler terms

That's mostly just me thinking aloud. Since it works for me regardless of the version I test against, I think it's just a temporary issue. But this should be handled more gracefully by not disconnecting the account when it happens.

**Comment by KamalGalrani** [comment]
The issue is repeating. It was there two days ago, it was there yesterday, it is there today and it is there even after I pulled master and recompiled.

**Comment by dgw** [comment]
I've just run into this as well. I last updated the plugin to 82e6bcf, but based on @KamalGalrani's comment it doesn't sound like running a more recent commit would have prevented the issue.

**Comment by dequis** [comment]
Confirmed and fixed with 59fcaab5096353bbe2f1366cbcf5f51290ca85ae (in the development branch)
